src/common/sockman.cpp

```diff
@@ -9,68 +9,77 @@
 #include <netbase.h>
 #include <util/sock.h>
 
-bool SockMan::BindListenPort(const CService& addrBind, bilingual_str& strError)
+bool SockMan::BindAndStartListening(const CService& to, bilingual_str& errmsg)
```

Comment:
4c23aacc2b82083f96255963f543dc35fff28334: since you need to print log statements here in a few places anyway, might as well drop the `errmsg` argument and always print logs here.